### PR TITLE
[minor] Make Manage's defaultJMS configurable

### PR DESCRIPTION
**Pull request by alequint**
Tested execution via pipeline in Fyre's pfvtsquadci cluster:

image

Related PRs:
- https://github.ibm.com/maximoappsuite/ansible-fvt/pull/276
- https://github.com/ibm-mas/cli/pull/461
- https://github.com/ibm-mas/ansible-devops/pull/1000

### CODE REVIEW

File: ibm/mas_devops/roles/suite_app_config/README.md [hunk 1]
```diff
@@ -294,6 +294,13 @@ Optional. Set this to `true` if you want to override existing Manage database en
 - Environment Variable: `MAS_APP_SETTINGS_OVERRIDE_ENCRYPTION_SECRETS_FLAG`
 - Default: `False`
 
+### mas_app_settings_default_jms
+Optional. Set this to `true` if you want to have JMS continuous queues configured
+
+- Environment Variable: `mas_app_settings_default_jms`
```
Comment (whitfiea) [inline ibm/mas_devops/roles/suite_app_config/README.md:300]: Should be uppercase?

**Review by whitfiea — CHANGES_REQUESTED** [review]

File: ibm/mas_devops/roles/suite_app_config/README.md (hunk 1 above)
Comment (alequint) [reply ibm/mas_devops/roles/suite_app_config/README.md:300]: Yes!

Comment (alequint) [reply ibm/mas_devops/roles/suite_app_config/README.md]: Done!

**Comment by alequint** [comment]
@andrercm @whitfiea could you please review it again and approve it if possible?

**Pull request merged**